crowbar_framework/config/routes.rb

```diff
@@ -123,5 +123,9 @@
   get "crowbar/:barclamp", action: "versions", controller: "barclamp"
   post "crowbar/:barclamp/1.0/:action/:id", controller: "barclamp"
 
+  get "installer", controller: "installer", action: "index", as: "installer"
```

Comment:
Maybe it's a pretty bad habit to write all the routes manually instead of just using something like

```
resource :installer, only: [:show] do
  collection do
    get :status
    get :install
  end
end
```

(The resource definition is just written out of my head without testing)